### ISSUE TITLE
Using planck in a webworker

### ISSUE THREAD

**Issue by zjeff222** [shakiba/planck.js#160]
I don't know anything about webpack but it seams to be blocking planck from loading into a web worker. There is an access to the window element, which is not defined in a worker, and it really seams to be webpack related.

**Comment by shakiba** [comment]
Might be related to this: https://github.com/webpack/webpack/issues/6525

**Comment by zjeff222** [comment]
Just to be clear, I was talking about the webpack code embeded in planck.js.  So I edited line 34 of the planck.js file.  Patching the library will do for me for now.  It's great to have box2d on javascript by the way :)

**Comment by shakiba** [comment]
Could you please try the solution described in [this](https://github.com/webpack/webpack/issues/6642#issuecomment-370222543) and [this](https://github.com/webpack/webpack/issues/6642#issuecomment-371087342) comments?

**Comment by zjeff222** [comment]
If I understand, this would be a solution where I am to use webpack.  I'm not using webpack.  So if I do understand, some webpack code is added to planck.js to make it better for those who use webpack ?  I only used 2-3 libraries up to now and it's the first one I see with embeded webpack code.

**Comment by shakiba** [comment]
Webpack combines source files and generates files like `planck.js` file, Planck distribution files are built with webpack. We need to configure webpack to fix the issue that you reported.

**Comment by shakiba** [comment]
Oh, what I meant by asking if that solution works for you was that if rebuilding planck with that solution will solve the problem. I did not mean to use that solution in your project.

**Comment by zjeff222** [comment]
Sorry, I don't want to learn how to build a js library at this time.  If you want to reproduce the problem, you only have to call a worker with `myWorker = new Worker('./worker.js');`
and in the worker.js import planck:  `importScripts('../../lib/planck.min.js');`

**Comment by shakiba** [comment]
Makes sense, no worries; if you need to build it for any other reason here are the commands (assuming git and npm are already installed):
```
git clone git@github.com:shakiba/planck.js.git
cd planck.js
npm install
npm run build
```
